website_sale_checkout_country_vat/static/src/js/dropdown.js

```diff
@@ -1,17 +1,19 @@
 /* Copyright 2016-2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+   Copyright 2019 Alexandre Díaz <jairo.llopis@tecnativa.com>
  * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
 odoo.define("website_sale_checkout_country_vat.dropdown", function (require) {
     "use strict";
-    var animation = require("web_editor.snippets.animation");
+    var animation = require('website.content.snippets.animation');
 
-    return animation.registry.website_sale_checkout_country_vat_dropdown
-    = animation.Class.extend({
-        selector: ".oe_website_sale:has(.js_country_dropdown, \
-                                        select[name=country_id])",
+    var CheckoutCountryVatDropdown = animation.Class.extend({
+        selector: ".oe_website_sale:has(.js_country_dropdown, " +
+                  "select[name=country_id])",
         start: function (editable_mode) {
             var result = this._super(editable_mode);
-            this.$address_country = this.$("select[name=country_id]");
-            this.$vat_no_country_field = this.$("#no_country_field");
+            this.$address_country = this.$target.find(
```

Comment:
I'm curious: why are you removing the `this.$` shorthand? It's available in all subclasses of `Widget`...

Reply:
Umm to ensure that doesn't have conflicts if the snippet it's used more than one time in the same page...

Reply:
Don't worry, it doesn't happen.

If you were to use `$(selector)`, yes, it's raw jquery and selects in the whole page, but `this.$()` is different and only searches inside the widget.